test/unit/server/web/auth/github.js

```diff
@@ -17,7 +17,7 @@ var MockAuthGitHub = {
 
     server.auth.strategy('github', 'MockGitHub');
     server.auth.strategy('session', 'cookie', {
-      password: 'testing'
+      password: 'password-should-be-32-characters'
```

Comment:
[new requirement of iron](https://github.com/hueniverse/iron/issues/39) which is a dependency of hapi